book/effects/http.md

```diff
@@ -156,28 +210,60 @@ update msg model =
           (Failure, Cmd.none)
 ```
 
+<!--
 Looking at the type signature, we see that we are not just returning an updated model. We are _also_ producing a **command** of what we want Elm to do.
+-->
+型シグネチャを見てみると、単に更新されたモデルが返されるだけでないことがわかります。ここでも _やはり_ Elmに実行して欲しい事を指示する**コマンド**を返します。
 
+<!--
 Moving on to the implementation, we pattern match on messages like normal. When a `GotText` message comes in, we inspect the `Result` of our HTTP request and update our model depending on whether it was a success or failure. The new part is that we also provide a command.
+-->
+実装では通常通りにメッセージに対してパターンマッチを行っています。`GotText`メッセージが来たときに、HTTPリクエストの`Result`の値を調べ、その結果が成功か失敗かによって我々のモデルを更新します。新しい部分は、ここでもやはりコマンドを返していることです。
 
+<!--
 So in the case that we got the full text successfully, we say `Cmd.none` to indicate that there is no more work to do. We already got the full text!
+-->
+つまり、全文の取得に成功した場合には`Cmd.none`としてこれ以上は何もしなくて良いことを示します。すでに全文を取得済みなのですから！
 
+<!--
 And in the case that there was some error, we also say `Cmd.none` and just give up. The text of the book did not load. If we wanted to get fancier, we could pattern match on the [`Http.Error`][Error] and retry the request if we got a timeout or something.
+-->
+そしてなんらかのエラーがあった場合には、やはり`Cmd.none`としてギブアップしてしまいます。本の内容を読み込むことができませんでしたので。もしより良い物にしたいのなら[`Http.Error`][Error]に対してパターンマッチを行い、タイムアウトなどのエラーであれば再度トライすることも可能です。
```

Comment:
> The text of the book did not load.
> 本の内容を読み込むことができませんでしたので。

* 英語で Because を省くことはかなりまれ
* 画面のステートに Loading があることから load を「画面に表示する」の意味で使っている可能性がある

の2点から、自信はないですが以下の訳のほうが適切かもしれません。

> その結果、画面上に本の内容は表示されません。